.docker/Dockerfile

```
@@ -11,10 +11,19 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   libleptonica-dev \
   zlib1g-dev \
   python3 \
+  python3-dev \
+  libssl-dev \
+  libffi-dev \
+  libjpeg-dev \
+  zlib1g-dev \
+  libqpdf-dev \
+  pybind11-dev \
+  libxml2-dev \ 
+  libxslt1-dev \
   python3-distutils \
   ca-certificates \
   curl \
-  git
+  git 
```

There shouldn't be a trailing white space character here
```suggestion
  git
```